### PR TITLE
Prepare 3.0.2 patch release

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## DO NOT SQUASH & MERGE 

Prepare 3.0.2 patch release

waiting on:

- [x] https://github.com/zowe/zowe-explorer-vscode/pull/3162
- [x] https://github.com/zowe/zowe-explorer-vscode/pull/3246
- [x] https://github.com/zowe/zowe-explorer-vscode/pull/3249
- [x] https://github.com/zowe/zowe-explorer-vscode/pull/3253
- [x] https://github.com/zowe/zowe-explorer-vscode/pull/3256

## Release Notes

Milestone: 3.0.2

Changelog: n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking change which adds or improves functionality)
- [ ] Breaking change (a change that would cause existing functionality to not work as expected)
- [ ] Documentation (Markdown, README updates)
- [x] Other (please specify above in "Proposed changes" section)

## Checklist

**General**

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/zowe-explorer-vscode/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] All PR dependencies have been merged and published (if applicable)
- [ ] A GIF or screenshot is included in the PR for visual changes
- [x] The pre-publish command has been executed:
  - **v2 and below:** `yarn workspace vscode-extension-for-zowe vscode:prepublish`
  - **v3:** `pnpm --filter vscode-extension-for-zowe vscode:prepublish`

**Code coverage**

- [ ] There is coverage for the code that I have added
- [ ] I have added new test cases and they are passing
- [ ] I have manually tested the changes

**Deployment**

- [ ] I have added developer documentation (if applicable)
- [ ] Documentation should be added to Zowe Docs
  - If you're an outside contributor, please post in the [#zowe-doc Slack channel](https://openmainframeproject.slack.com/archives/CC961JYMQ) to coordinate documentation.
  - Otherwise, please check with the rest of the squad about any needed documentation before merging.
- [ ] These changes may need ported to the appropriate branches (list here):

### CODE REVIEW

File: packages/zowe-explorer-api/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish.
```
Comment (anaxceron) [inline packages/zowe-explorer-api/CHANGELOG.md:9]: Could we add an issue or PR link to this?

File: packages/zowe-explorer/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 ### New features and enhancements
 
+- Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish. Download the respective language pack and switch.
```
Comment (anaxceron) [inline packages/zowe-explorer/CHANGELOG.md:9]: Same comment as the first.

**Review by anaxceron — APPROVED** [review]
Looks good to me. Made suggestions to include issue/pr links w/ changelog items.

**Review by likhithanimma1 — APPROVED** [review]

**Review by likhithanimma1 — APPROVED** [review]

File: packages/zowe-explorer/package.json [hunk 3]
```diff
@@ -1768,7 +1768,7 @@
     ]
   },
   "scripts": {
-    "build": "pnpm copy-secrets && pnpm clean:bundle && pnpm license && webpack --mode development && pnpm madge",
+    "build": "pnpm copy-secrets && pnpm clean:bundle && pnpm license  && pnpm build:nls && webpack --mode development && pnpm madge",
```
Comment (traeok) [inline packages/zowe-explorer/package.json:1771]: not requesting changes, but there's an extra space after `pnpm license`

**Review by traeok — CHANGES_REQUESTED** [review]
Seeing a bug with convert v1 profiles. Looking into a fix

File: packages/zowe-explorer/package.json (hunk 3 above)
Comment (JillieBeanSim) [reply packages/zowe-explorer/package.json:1771]: @traeok can you explain the issue seen please? was it brought in with previous changes or pre-existing with 3.0.1? I will push a fix for the extra space but if the bug is existing in 3.0.1 it should wait until the 3.0.3 patch release.

**Review by traeok — APPROVED** [review]

**Review by t1m0thyj — APPROVED** [review]
LGTM, thanks @JillieBeanSim for doing all the cherry picks 😁

**Comment by JillieBeanSim** [comment]
Thanks all, I will be do a Squash & merge. If I change to just merge we get warning stating the branch wants linear history

**Pull request merged**